source/server/lsp-servers.adb

```diff
@@ -993,9 +993,9 @@ package body LSP.Servers is
                Last : Natural := 0;
 
             begin
-               for E in Vector.Each_Stream_Element loop
+               for E of Vector loop
                   Last := Last + 1;
-                  Aux (Last) := Character'Val (E.Element);
+                  Aux (Last) := Character'Val (E);
                end loop;
 
                Server.Out_Trace.Trace (Aux (Aux'First .. Last));
```

Comment:
I suppose result of VSS.Stream_Element_Vectors.Conversions.Unchecked_To_String function can be passed directly here, without intermediate buffer.

Use of Last looks strange here, isn't Last = Vector.Length here?

Reply:
A good idea, thanks